src/lazyload-image.directive.ts

```diff
@@ -69,7 +69,7 @@ export class LazyLoadImageDirective implements OnChanges, AfterContentInit, OnDe
           tap(attributes => attributes.onStateChange.emit({ reason: 'setup' })),
           tap(attributes => this.hooks.setup(attributes)),
           switchMap(attributes => {
-            if (!attributes.imagePath) {
+            if (!attributes.imagePath && isImageElement(attributes.element)) {
```

Comment:
So is the issue you're bringing up here that the default image wouldn't be displayed correctly in this case with the async image? Because other than that, I can't see why it would be a problem to simply remove the inline background definition if it's an empty string...

The trick here is not to look at this like we're 'using the path from css' so much as look at it like we're 'removing the inline background definition', whether someone has defined a background in CSS is their own choice.

Reply:
I agree that this is an edge case and maybe not a problem but take this example:
We have a function to decide what image we should load:
```ts
async getImagePath() {
  if (something) {
    return 'https://path/to/image';
  } else {
    return ''; // This means we should use the image from css (?)
  }
}
```
And then: `<img [lazyLoad]="getImagePath() | async">`

In this case, will the image stated in the CSS always be loaded since `lazyLoad` will be falsy at first.